modules/apps/search/search-web/src/META-INF/resources/view.jsp

```diff
@@ -57,7 +57,13 @@ pageContext.setAttribute("portletURL", portletURL);
 		</aui:select>
 
 		<aui:field-wrapper inlineField="<%= true %>">
-			<aui:button cssClass="search-button" icon="icon-search" onClick='<%= renderResponse.getNamespace() + "search();" %>' value="search" />
+			<a class="btn btn-default" href="javascript:;" onclick="<%= renderResponse.getNamespace() + "search();" %>">
```

Comment:
Could we add a span into a button?

<button type="button">
<i class="icon-search"></i> 
<span>Search</span>
</button>

We need hide the text "Search" but not the icon.